api/REST-API.md

```diff
@@ -1,90 +1,94 @@
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 10 min
+- Guided Practice: 30 min
+- Independent Practice: 30 min
+- Check for Understanding: 5 min
 
 ### Prerequisites
 
 Here are links to lessons that should be completed before this lesson:
+- [API/JSON Lesson](/api/apis-and-json.md)
 
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+REST completely changed software engineering after 2000. This new approach to developing web projects and services was defined by Roy Fielding, father of the HTTP specification and one of the leading international authorities on everything to do with Network Architecture, in his dissertation entitled "Architectural Styles and the Design of Network-based Software Architectures". In the field of APIs, REST (Representational State Transfer) is today the "be all and end all" in service app development.
+Today there are no projects or applications that don't have a REST API for the creation of professional services based on this software. Twitter, YouTube, Facebook identification systems… hundreds of companies generate business thanks to REST and REST APIs. Without them any horizontal growth would be practically impossible. This is because REST is the most logical, efficient and widespread standard in the creation of APIs for Internet services.
 
 ### Objectives
 
-**Participants will be able to:**
+- What is REST?
+- REST Features.
+- Guiding Principles of REST.
+- HTTP Methods
+- Advantages of REST
 
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
-
-### Specific Things To Teach
-
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
 
 ### Materials
-
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
-
+- [Here is the link to Slideshow](https://docs.google.com/presentation/d/1ZdnhhGbwyJcmgPnTn6eeHjdhekUvo0ht70MwC7Ll5zw/edit#slide=id.g54f3514d6b_1_63)
+- [Representational Rest API](https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm)
+- [REST API Tutorial](RestApiTutorial.com )
+- [Creating a Secure REST Api in Nodejs](https://www.toptal.com/nodejs/secure-rest-api-in-nodejs)
+- [Tutorial on Rest API](https://www.tutorialspoint.com/restful/index.htm)
+ 
```

Comment:
What part of this are they supposed to read? This site leads to a Java API tutorial.

Reply:
Please address

Reply:
sorry I couldn't understand. Please, can you explain what I need to explain?

Reply:
If you are assigning this resource, what are you actually assigning? Are you assigning all 30 pages? The first 2 pages? What do you expect, and how long will it take?

Reply:
Tell them which parts of these links (besides the slides) that they should be doing, and how long they should spend on it. Otherwise they could think you expect them to look through the entire site thoroughly.

Reply:
ohk sure.